lib/electron/index.js

```diff
@@ -111,25 +112,43 @@ class ElectronRunner {
       })
     }).then(() => {
       return new Promise((resolve, reject) => {
-        log(`Packaging app with Electron packager...`)
+        log(`Packaging app with Electron ${builderConfig !== undefined ? 'builder' : 'packager'}...`)
         log()
 
-        const packager = require(appPaths.resolve.app('node_modules/electron-packager'))
-        packager(packagerConfig, (err, appPaths) => {
-          log()
-
-          if (err) {
-            warn(`[FAIL] Electron packager could not build`)
-            console.log(err + '\n')
-            reject()
-            return
-          }
+        if (builderConfig !== undefined) {
+          const builder = require(appPaths.resolve.app('node_modules/electron-builder'))
+          builder.build({ config: builderConfig })
```

Comment:
Assume that current working directory equals to project dir, right?

Reply:
Hmm... I've missed to include some defaults in `quasar-config.js`. Please take a look in the next commits and tell me if they are the correct analogous to electron-packager settings.